tests/util/test_yaml.py

```diff
@@ -121,18 +134,25 @@ def test_include_dir_named(self, mock_walk):
     def test_include_dir_named_recursive(self, mock_walk):
         """Test include dir named yaml."""
         mock_walk.return_value = [
-            ['/tmp', ['tmp2'], ['first.yaml']],
+            ['/tmp', ['tmp2', '.ignore', 'ignore'], ['first.yaml']],
             ['/tmp/tmp2', [], ['second.yaml', 'third.yaml']],
+            ['/tmp/.ignore', [], []],
```

Comment:
I'm not sure `os.walk` will return this at all (but at least the two dicts are empty, so should not cause an issue